tests/test_python_interface.py

```diff
@@ -50,24 +50,35 @@ def test_variables_of_different_types(description, identifier, variables, expect
     assert result == expected
 
 
-def test_large_python_integers_fail_sensibly():
+@pytest.mark.parametrize(
+    "key",
+    (
+        object(),
+        34.3,
+        1_000_000_000_000,  # Larger than unsigned long integer.
```

Comment:
Suggestion: It looks like ints are unacceptable, regardless of if they're in range. Perhaps we should add that to the test too.

(Perhaps it's just not clear to me why we're testing out-of-range values in this way.)

Reply:
Ints are acceptable if they are in the range of signed long integers. (Oops, this should be 'signed', not 'unsigned' - I'll correct that before merging.)

It's to do with the crossover between Python integers (which can be any size) and Rust (which are statically determined). In practice it's unlikely that someone is going to want to pass a huge integer to the translation system but I wanted to call it out as a potential behaviour for completeness.

Reply:
Oh that's a good point - the out of range thing is only relevant for the variable values. I'll change it.